pkgs/applications/video/dvdbackup/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, fetchurl, libdvdread, libdvdcss, dvdauthor }:
+
+stdenv.mkDerivation rec {
+  version = "0.4.2";
+  name = "dvdbackup-${version}";
+
+  src = fetchurl {
+    url = "http://downloads.sourceforge.net/dvdbackup/dvdbackup-${version}.tar.xz";
```

Comment:
`fetchurl` supports sourceforge mirrors. Something like `mirror://sourceforge/dvdbackup/${name}.tar.xz` should work.